README.md

```diff
@@ -51,6 +51,16 @@ Windows)*:
 $ pip install tensorflow-gpu
 ```
 
+## Upgrade Tensorflow
+
+
+To update Tensorflow to the latest version(currently 2.0)
```

Comment:
NIT: 
```suggestion
To update Tensorflow to the latest version (currently 2.0)
```

Reply:
There is no major changes.

Reply:
Changes as per new request of @mihaimaruseac  is done please review my PR.